From 5ef13e96bdb71cf2ebda2c3732b04748eea21721 Mon Sep 17 00:00:00 2001
From: Timothy Arceri <tarceri@itsqueeze.com>
Date: Wed, 8 Mar 2017 14:36:14 +1100
Subject: [PATCH 26/33] gallium/util: remove unused util_memmove()

This is not used anywhere and Visual Studio looks to have
supported memmove() for a long time if not always.
---
 src/gallium/auxiliary/util/u_string.h | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/src/gallium/auxiliary/util/u_string.h b/src/gallium/auxiliary/util/u_string.h
index adcdf200ec..9765aaa014 100644
--- a/src/gallium/auxiliary/util/u_string.h
+++ b/src/gallium/auxiliary/util/u_string.h
@@ -179,25 +179,6 @@ util_strstr(const char *haystack, const char *needle)
    return NULL;
 }
 
-static inline void *
-util_memmove(void *dest, const void *src, size_t n)
-{
-   char *p = (char *)dest;
-   const char *q = (const char *)src;
-   if (dest < src) {
-      while (n--)
-	 *p++ = *q++;
-   }
-   else
-   {
-      p += n;
-      q += n;
-      while (n--)
-	 *--p = *--q;
-   }
-   return dest;
-}
-
 
 #define util_strcasecmp stricmp
 
@@ -212,7 +193,6 @@ util_memmove(void *dest, const void *src, size_t n)
 #define util_strncmp strncmp
 #define util_strncat strncat
 #define util_strstr strstr
-#define util_memmove memmove
 #define util_strcasecmp strcasecmp
 
 #endif
-- 
2.12.0

